components/experimental/src/units/converter.rs

```diff
@@ -68,14 +67,14 @@ pub(crate) struct OffsetConverter {
     pub(crate) proportional: ProportionalConverter,
 
     /// The offset value to be added to the result of the proportional converter.
-    pub(crate) offset: Ratio<BigInt>,
+    pub(crate) offset: IcuRatio,
 }
 
 impl OffsetConverter {
     /// Converts the given value from the input unit to the output unit.
-    pub(crate) fn convert(&self, value: &Ratio<BigInt>) -> Ratio<BigInt> {
+    pub(crate) fn convert(&self, value: &IcuRatio) -> IcuRatio {
         let result = self.proportional.convert(value);
-        result + &self.offset
+        result + self.offset.clone()
```

Comment:
Issue: unnecessary cloning

Reply:
good catch.

I need to implement this too:

```
impl Add<&IcuRatio> for &IcuRatio {
    type Output = IcuRatio;

    fn add(self, rhs: &IcuRatio) -> IcuRatio {
        IcuRatio(&self.0 + &rhs.0)
    }
}
```